torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -2731,6 +2731,16 @@ class TestPickler:
             torch.distributed.rpc.api._default_pickler is _internal_rpc_pickler
         )
 
+    @dist_init
+    def test_wait_all(self):
+        with rpc.wait_all():
+            self.assertTrue(rpc.thread_local_var.future_list == [])
+            dst = worker_name((self.rank + 1) % self.world_size)
+            fut = rpc.rpc_async(dst, torch.add, (torch.ones(2, 2), 1))
```

Comment:
@mrshenli I presume it would be preferred to check that the only element in future_list is a C futures object here.  If you agree, how would I do that?

Reply:
You can try `isinstance(rpc.thread_local_var.future_list[0], torch._C.Future)`